Cargo.toml

```diff
@@ -2,6 +2,9 @@
 members = [ "harper-cli", "harper-core", "harper-ls", "harper-comments", "harper-wasm", "harper-tree-sitter", "harper-html", "harper-literate-haskell", "harper-typst" , "harper-stats"]
 resolver = "2"
 
+[profile.test]
+opt-level = 1
```

Comment:
I really think the issues with debug breakpoints in tests not working should be documented. The first time, I hit this issue, it took me a while to figure out.